client/app/bundles/course/video/components/tables/VideoTable.tsx

```diff
@@ -76,7 +91,7 @@ const VideoTable: FC<Props> = (props) => {
       label: 'Title',
       options: {
         filter: false,
-        sort: false,
+        sort: true,
```

Comment:
Is this needed?

Reply:
actually it's not included in the requirements, but I saw the majority of other tables (Forums, Manage Users) have their first column after S/N to be sortable. However, survey does not. So I'm not really sure. I think for now, I can revert this behavior